src/w3c/templates/cgbg-headers.js

```diff
@@ -49,6 +49,10 @@ export default conf => {
     ` : ""}
     <dt>${conf.multipleEditors ? html`${conf.l10n.editors}` : html`${conf.l10n.editor}`}</dt>
     ${showPeople(conf, "Editor", conf.editors)}
+    ${conf.formerEditors ? html`
+      <dt>${conf.multipleFormerEditors ? html`${conf.l10n.former_editors}` : html`${conf.l10n.former_editor}`}</dt>
```

Comment:
nit:
```HTML
<dt>${conf.multipleFormerEditors ? conf.l10n.former_editors : conf.l10n.former_editor}</dt>
```
These are always a string, not HTML.

Reply:
did that based on code above (like for `multipleEditors`). will fix them also.